recipes-devtools/clang/clang/0026-For-x86_64-set-Yocto-based-GCC-install-search-path.patch

```diff
@@ -59,7 +64,7 @@ index 05d1d3003881..e56812145e6d 100644
        "x86_64-redhat-linux",    "x86_64-suse-linux",
        "x86_64-manbo-linux-gnu", "x86_64-linux-gnu",
        "x86_64-slackware-linux", "x86_64-unknown-linux",
-+      "x86_64-oe-linux", "x86_64-poky-linux", "x86_64-wrs-linux",
++      "x86_64-oe-linux", CLANG_EXTRA_OE_VENDORS_TRIPLES
```

Comment:
Lets make it `//CLANG_EXTRA_OE_VENDORS_TRIPLES` so the code can keep compiling outside OE builds.